src/components/PlaybackControls/index.tsx

```diff
@@ -148,8 +120,7 @@ const PlayBackControls = ({
             </div>
             <Slider
                 value={time}
-                onChange={handleTimeChange}
-                onAfterChange={handleSliderMouseUp}
+                onChange={onTimeChange}
```

Comment:
Hmm... while I agree that this change does fix the buffering issue, as a user I would expect playback to pause while I'm interacting with the slider. This might be a good thing to get confirmation from UX on.

Ant now has an `onChangeComplete` instead of `onAfterChange`... could that fix some of the issues? https://ant.design/components/slider